internal/testutil/mig/pre_check.go

```diff
@@ -22,3 +22,9 @@ func PreCheckBasicOwnerID(tb testing.TB) {
 	tb.Helper()
 	PreCheckBasic(tb)
 }
+
+func PreCheckAtlasUsername(tb testing.TB) {
+	tb.Helper()
+	checkAtlasUser(tb)
+	PreCheckBasic(tb)
+}
```

Comment:
the usual way to implement mig precheck is to call the acc one, e.g.: 
func PreCheckAtlasUsername(tb testing.TB) {
	tb.Helper()
	checkLastVersion(tb)
	acc. PreCheckAtlasUsername(tb)
}

and you can delete checkAtlasUser from mig

Reply:
Thanks! Done